### DISCUSSION TITLE
ghippo/user-guide/access-control/docking/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#5701] [Announcements]
# ghippo/user-guide/access-control/docking/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/ghippo/user-guide/access-control/docking/

**Comment by panpan0000** [comment]
这里的（2）“在 创建 SSO 接入 页面填写客户端 ID。”
这里表单里的“客户端访问地址”是什么意思？文档没有解释。是keycloak 服务器的外部访问地址么？比如ghippo-keycloakx-http这个svc的外部NodePort地址？

**Reply by panpan0000** [reply]
而且（3）创建 SSO 接入成功后，在 接入管理 管理列表，点击刚创建的客户端 ID 进入详情， 复制客户端 ID、密钥和单点登录 URL 信息，填写至客户系统完成用户体系打通。


这里应该是第二套DCE（一套作为OIDC的源/服务端，另一套DCE作为OIDC的客户端）上进行的操作，对么

**Reply by panpan0000** [reply]
请忽略对（3）的评价和疑问

**Reply by panpan0000** [reply]
创建一个新的SSO接入，是创建一个新的realm么？还是在已有realm新加一个client？
而且创建新client需要填写Redirect Url ，界面表单上也没有地方填呀？
如果是创建一个新的OIDC的client，那么客户端（比如OEM-IN）需要知道 ： OIDC服务器的URL， realm名字， client名字，client secret 。

但是这些在接入管理里缺少Realm和URL

**Reply by lemontrees123** [reply]
1. 客户端访问地址：用户完成登录并通过身份验证后，认证服务器用来重定向用户的地址，即 Callback URL；2. Redirect Url对应的是客户端访问地址，即Callback URL；3. realm名字：ghippo；4. client名字:客户端ID；5. client secret ：密钥
